lib/services/database.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+import '../data.dart';
```

Comment:
I suggest adding the code directly to the data.dart file.

Reply:
I suggest splitting this data.dart into separate files with isolated logic instead of using this data.dart as a scrapyard for everything